### DISCUSSION TITLE
Can snapshot be used to restore the cluster?

### DISCUSSION THREAD

**Discussion by ao-song** [tikv/client-rust#335] [Q&A] [answered]
Hi,

I found the snapshot in tnx client, seems I can use it to read data at that time, but can I use this snapshot to restore the cluster somehow or how can I restore the cluster to some previous checkpoints via the client? Thanks!

Brs, Ao

**Comment by ekexium** [comment, marked as answer]
client-rust itself doesn't support backup and restoration. You can check other tools (for example [BR](https://docs.pingcap.com/tidb/stable/backup-and-restore-tool), I think). @andylokandy any best practice?

I don't suggest using the snapshot to do this for effeciency and reliability concerns.

**Reply by ao-song** [reply]
I see, thanks!
